docs/index.Rmd

```diff
@@ -567,8 +583,17 @@ Grouping can be done by:
 
 * single column name
 * seq of column names
-* map of keys (group names) and row indexes
 * value returned by function taking row as map (limited to `:select-keys`)
+* map of keys (arbitrary group names) to sequences of row indexes
+
+In the case of the first three of these methods, each sub-dataset contains all and only rows from the original data set that share the same grouping value:
+
+* the value of the row in a specified single column
+* the sequence of resulting values for a sequence of column names
```

Comment:
In this case the result is a map not a sequence.

```
(tc/group-by DS [:V1])
;; => _unnamed [2 3]:
;;    |   :name | :group-id |                 :data |
;;    |---------|----------:|-----------------------|
;;    | {:V1 1} |         0 | Group: {:V1 1} [5 4]: |
;;    | {:V1 2} |         1 | Group: {:V1 2} [4 4]: |
```

Reply:
Ah, I didn't realize that.  I'll fix it.